src/quadratic.jl

```diff
@@ -0,0 +1,66 @@
+"""
+Helper function for quadratic_interval
+"""
+function quadratic_helper(a::Number, b::Number, c::Number, L::AbstractArray)
```

Comment:
If this modifies `L`, then it should be 

```
quadratic_helper!(L, a, b, c)
```

Reply:
These two functions implement a non-obvious algorithm, so I would suggest adding at least some hints as to what's going on, even though it's explained in the book.